playbooks/roles/wordpress/tasks/nginx.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: open HTTPS port on the firewall
+  ufw:
+    rule: allow
+    port: "{{ item }}"
+    proto: tcp
+  with_items:
+    - 443
+
+- name: mount /var/www
+  mount: name=/var/www src=/dev/vdb1 fstype=ext4 state=mounted
+
+- name: chown www-data /var/www
+  file: path=/var/www mode=750 owner=www-data group=www-data state=directory
+
+- name: mkdir /var/www/log
+  file: path=/var/www/log mode=750 state=directory
+
+- name: Ensure directory for config files exists
+  file:
+    path: /etc/nginx/sites-available/wordpress
+    state: directory
+
+- name: create x template
+  template:
+    src: "{{ item }}"
+    dest: /etc/nginx/sites-available/wordpress/{{ item | basename | regex_replace('-nginx\.j2','.conf') }}
+  with_fileglob:
+    - ../templates/*-nginx.j2
+
+- name: Find needed nginx configurations
+  find:
+    paths: /etc/nginx/sites-available/wordpress
+  register: found
+
+- name: Enable nginx site configurations
+  file:
+    src: "{{ item.path }}"
+    dest: /etc/nginx/sites-enabled/{{ item.path | basename }}
```

Comment:
@Agrendalath, can we name the configuration files `<site domain name>.conf`?

Reply:
Yup, that makes sense. Done.